packages/db-objects-redis/src/lib/objects/objectsInRedisClient.ts

```diff
@@ -1226,12 +1216,16 @@ export class ObjectsInRedisClient {
         }
 
         options = options || {};
-        this.checkFileRights(id, name, options, CONSTS.ACCESS_READ, (err, options, meta) => {
+        this.checkFileRights(id, name, options, CONSTS.ACCESS_READ, async (err, options, meta) => {
             if (err) {
                 return tools.maybeCallbackWithError(callback, err);
             } else {
-                // @ts-expect-error if callback would be falsy, we would already have returned above
```

Comment:
The explanation still holds true. I was confused at first that you'd try to return a Promise from a nested callback. might be worth keeping.